package.json

```diff
@@ -67,6 +67,11 @@
     "nsis": {
       "perMachine": true
     },
+    "mac": {
+      "extendInfo": {
+				"LSUIElement": 1
```

Comment:
Can you change the tabs used here to spaces?

Reply:
done :)